dataflow/src/main/java/org/checkerframework/dataflow/expression/FieldAccess.java

```diff
@@ -11,14 +11,30 @@
 import org.checkerframework.javacutil.ElementUtils;
 import org.checkerframework.javacutil.TypesUtils;
 
+/**
+ * A FieldAccess represents a field access or the occurrence of {@code .class} in source code, e.g.,
+ * SomeClass.class, SomeClass[].class, int.class, or int[].class.
```

Comment:
I thought `ClassName` represents class literals?

Reply:
In the current code, it sometimes does and sometimes does not.  I guessed wrong about how to rectify the inconsistency...